src/core/eme/types.ts

```diff
@@ -179,9 +179,30 @@ export type IEMEManagerEvent = IEMEWarningEvent | // minor error
 export type ILicense = BufferSource |
                        ArrayBuffer;
 
-// Segment protection manually sent to the EMEManager
-export interface IContentProtection { type : string; // initDataType
-                                      data : Uint8Array; } // initData
+/** Segment protection sent by the RxPlayer to the EMEManager. */
+export interface IContentProtection {
```

Comment:
I don't remember, have we discussed the fact that this object could have a getter that returns the concatenated data ?

Reply:
Yes in another comment:

---

 grenault73 20 days ago

I was wondering, as there are some places in the code where the concatenated init data is needed, we could maybe integrate a function in the initializationData that returns it, no?

---

peaBerberian 18 days ago

Actually there are two places:
  - here, because ultimately the generateRequest EME API requires it under that format
  - in the PersistentSessionsStore, because that concatenation's has was previously used as an ID to identify persisted sessions.

So they are needed at two places but for two very different reasons IMO. For example the first could be in any order whereas the second has to follow a specific order.

As such, I don't know if I want to use a common function here. Being explicit in the PersistentSessionsStore may be clearer.